README.md

```diff
@@ -48,9 +48,7 @@ introduced in Solidus v2.4. This maps better to how the TaxJar API itself works.
     [Configuration](#configuration).
 
 1. Finally, make sure that the `TAXJAR_API_KEY` environment variable is set to
-  your TaxJar API key and make sure that you have a `Spree::TaxRate` with the name
-  "Sales Tax". This will be used as the source for the tax adjustments that
-  Solidus creates.
```

Comment:
Could we change this to something like 
```
The extension looks for a `Spree::TaxRate` with the name "<NAME>", and if one is not found it will be created. This will be used as the source for the tax adjustments that Solidus creates.
```
I think documenting what tax rate the extension uses in the README is valuable information.

Reply:
I'm gonna make a wiki page regarding tax rates and move that information there in an effort to consolidate new documentation there!